libs/template/templates/default-python/template/{{.project_name}}/fixtures/.gitkeep.tmpl

```diff
@@ -0,0 +1,17 @@
+# Fixtures
+{{- /*
+We don't want to have too many README.md files, since they
+stand out so much. But we do need to have a file here to make
+sure the folder is added to Git.
+*/}}
+
+This folder is reserved for fixtures, such as csv files.
+
+Below is an example of how to load fixtures as a data frame:
+
+```
+import pandas as pd
+
+df = pd.read_csv("../fixtures/mycsv.csv")
+display(df)
+```
```

Comment:
Loading a fixture with a relative path requires us to set $PWD.

Reply:
Ah yes there was supposed to be some path mangling here. Which is not required on serverless, btw.

I don't think there's any way we can do this without the (broadly published) .entry_point API of dbutils. Or is there, @fjakobs?

This is what we could suggest:

```
import pandas as pd
import os

def get_relative_path(*relative_parts):
    base_dir = os.path.dirname(dbutils.notebook.entry_point.getDbutils().notebook().getContext().notebookPath().get())
    path = os.path.normpath(os.path.join(base_dir, *relative_parts))
    return path if path.startswith("/Workspace") else os.path.join("/Workspace", path)

csv_file = get_relative_path("..", "fixtures", "mycsv.csv")
df = pd.read_csv(csv_file)
display(df)
```

Reply:
The IDE adds some bootstrap code that sets CWD and python path correctly. We were considering to do the same for code deployed through DABs

Reply:
@fjakobs We need something that also works from a notebook in the workspace, though. Is there anything cleaner for that than the option above?

And, as we get to a solution here, we should look at moving this to public docs, e.g. to https://docs.databricks.com/en/external-data/csv.html. cc @PaulCornellDB